src/pages/[platform]/build-a-backend/more-features/analytics/analytics-migration-guide/index.mdx

```diff
@@ -26,110 +26,492 @@ export function getStaticProps(context) {
--   attributes: {
--     attr: 'attr'
--   },
--   // OPTIONAL, the service provider, by default is the Amazon Pinpoint
--   provider: 'AWSPinpoint'
-- });
-
-+ configureAutoTrack({
-+   // REQUIRED, turn on/off the auto tracking
-+   enable: true,
-+   // REQUIRED, the event type, it's one of 'event', 'pageView' or 'session'
-+   type: 'session',
-+   // OPTIONAL, additional options for the tracked event.
-+   options: {
-+     // OPTIONAL, the attributes of the event
-+     attributes: {
-+       customizableField: 'attr'
-+     }
-+   }
-+ });
-```
-## Analytics.identifyUser
-
-You can identify an Analytics user with the `identifyUser` API, 
-
-```ts
-import { identifyUser } from 'aws-amplify/analytics';
-
-identifyUser({
-  userId: "<user-id>", // If the user was signed in through Amplify Auth's signIn method you can retrieve the current user's ID from there.
-  userProfile
-});
-
-```
-## Streaming data
-You can stream analytics data to Kinesis. To configure Kinesis with Amplify, reference the [Installation and Configuration](https://docs.amplify.aws/react/build-a-backend/more-features/analytics/streaming-data/#installation-and-configuration) section of the documentation. Below is an example on how to stream data and flush events,
-
-```ts
-import { record, flushEvents } from 'aws-amplify/analytics/kinesis';
-
-record({
-  data: {
-    // The data blob to put into the record
-  },
-  partitionKey: 'myPartitionKey',
-  streamName: 'myKinesisStream'
-});
-
-flushEvents();
-```
-Finally, let's look at the difference in how to personalize recommendations,
-
-```diff
-- import { Analytics, AmazonPersonalizeProvider } from 'aws-amplify';
-- Analytics.addPluggable(new AmazonPersonalizeProvider());
-
-+ import { record } from 'aws-amplify/analytics/personalize';
-
-
-- Analytics.record({
--     eventType: "Identify",
--     properties: {
--         "userId": "<USER_ID>"
--     }
-- }, 'AmazonPersonalize');
-
-+ record({
-+   eventType: "Identify",
-+   properties: {
-+      userId: "<USER_ID>"
-+   }
-+ });
-
-```
-Notice that in v5, the `Analytics.record` API takes in a second parameter with the string value `AmazonPersonalize` whereas in v6, the `record` function is imported from the `aws-amplify/analytics/personalize` subpath and takes only one parameter. 
+This guide will help you migrate Amplify JavaScript v5's Analytics APIs to the new v6's Analytics APIs.
+
+As of v6 of Amplify, you will now import APIs directly from the `aws-amplify/analytics` path when using the Pinpoint provider. If you are using the Personalize, Kinesis, or Kinesis (Stream) provider, you will need to import separate APIs from the appropriate subpath (ie `import { record } from 'aws-amplify/analytics/personalize'` when using the `record` API with the `AmazonPersonalize` provider) instead of passing in the `provider` or `options.provider` parameter.
+
+<Callout>TODO: The `startSession` API has been removed in v6. (Due to auto-track capabilities?)</Callout>
```

Comment:
What's the status on this?

Reply:
Waiting on some team input here, will update when I get some clarification.